content/en/contribute/medic/onboarding/all-the-things.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "All The Things"
+linkTitle: "All The Things"
+weight: 2
+description: >
+  Wide range of topics and resources to support the onboarding
+---
+
+This page is meant to serve as a point of conversation, with a wide range of topics to be discussed when joining Medic or starting as a contributor. Many things are not in any particular order. The goal is to convey a general “lay of the land” so someone starting can see a lot of what’s out there without having to be surprised each day as new things pop up.
+
+## First, some general context…
+
+### What is the CHT?
+* Why the [CHT (Community Health Toolkit)]({{% ref "/why-the-cht" %}})?
+* The [CHT Core Framework]({{% ref "/core" %}}) - Web application that runs inside an Android app (as a [WebView](https://developer.android.com/reference/android/webkit/WebView))
```

Comment:
As first timer, this webview link is confusing because it doesn't explain much (it's a class definition), and now we also have PWA

```suggestion
* The [CHT Core Framework]({{% ref "/core" %}}) - App that can be accessed in the [browser](https://docs.communityhealthtoolkit.org/core/overview/architecture/#cht-web-application) or as [PWA](https://docs.communityhealthtoolkit.org/core/overview/pwa/) or [native Android app](https://docs.communityhealthtoolkit.org/core/overview/architecture/#cht-android).
```